tests/phpunit/tests/blocks/renderCommentTemplate.php

```diff
@@ -348,4 +378,53 @@ function test_build_comment_query_vars_from_block_with_comment_preview() {
 			)
 		);
 	}
+
+	/**
+	 * Test rendering an unapproved comment preview.
+	 */
+	function test_rendering_comment_template_unmoderated_preview() {
+		$parsed_blocks = parse_blocks(
+			'<!-- wp:comment-template --><!-- wp:comment-author-name /--><!-- wp:comment-content /--><!-- /wp:comment-template -->'
+		);
+
+		$unapproved_comment = self::factory()->comment->create_post_comments(
+			self::$custom_post->ID,
+			1,
+			array(
+				'comment_author'       => 'Visitor',
+				'comment_author_email' => 'unapproved@example.org',
+				'comment_author_url'   => 'http://example.com/unapproved/',
+				'comment_content'      => 'Hi there! My comment needs moderation.',
+				'comment_approved'     => 0,
+			)
+		);
+
+		$block = new WP_Block(
+			$parsed_blocks[0],
+			array(
+				'postId' => self::$custom_post->ID,
+			)
+		);
+
+		$commenter_filter = function () {
+			return array(
+				'comment_author_email' => 'unapproved@example.org',
+			);
+		};
```

Comment:
Micro-optimization - use static closure for a tiny performance improvement.

```suggestion
		$commenter_filter = static function () {
			return array(
				'comment_author_email' => 'unapproved@example.org',
			);
		};
```